### PR TITLE
Fixes certain material containers

### LINKED ISSUE #33321: Protolathes and Exosuit Fabricators do not appear to accept materials
This is replicable as of 917572c, with 85ddf6b and 418f9ed testmerged.

Using in-hand materials on protolathes and exosuit fabricators does not result in the materials being consumed by the machine.

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #33321

I just realized every material container sets max_amount to 0 and still works anyway. UMMMMMMMM.....

### CODE REVIEW

File: code/_compile_options.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #define DEBUG					//Enables byond profiling and full runtime logs - note, this may also be defined in your .dme file
 								//Enables in-depth debug messages to runtime log (used for debugging)
-//#define TESTING				//By using the testing("message") proc you can create debug-feedback for people with this
+#define TESTING				//By using the testing("message") proc you can create debug-feedback for people with this
```
Comment (praisenarsie) [inline code/_compile_options.dm]: 🚶

**Comment by Thunder12345** [comment]
@Cyberboss You just turned testing on for the live servers

**Comment by Cyberboss** [comment]
Round didn't start, I plead statute of limitations

**Comment by davethwave** [comment]
Kevinz000 seems to have fixed it though the pr hasn't been merged yet. https://github.com/tgstation/tgstation/pull/33393

**Pull request merged**